stories/module-adsense-settings.stories.js

```diff
@@ -43,14 +43,14 @@ import {
 	ACCOUNT_STATUS_APPROVED,
 	SITE_STATUS_ADDED,
 } from '../assets/js/modules/adsense/util/status';
-import { WithTestRegistry } from '../tests/js/utils';
+import createLegacySettingsWrapper from './utils/create-legacy-settings-wrapper';
 
 function filterAdSenseSettings() {
 	removeAllFilters( 'googlesitekit.ModuleSettingsDetails-adsense' );
 	addFilter(
 		'googlesitekit.ModuleSettingsDetails-adsense',
 		'googlesitekit.AdSenseModuleSettings',
-		fillFilterWithComponent( SettingsMain )
+		fillFilterWithComponent( AdSenseSettings )
 	);
 }
```

Comment:
This wasn't explicitly mentioned in the implementation brief, but anything like this which was previously used with the old `Settings` component is no longer necessary because it's built into the new `SettingsLegacy` wrapper and can be removed. Most other modules have a similar `filter{Module}Settings` function. Removing this will likely remove the need for a few imports as well.

Same below for `filterAdSenseSettingsSetupIncomplete` as this is controlled by the datastore now which is put into this state by each story, so this can be removed too.